vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -351,6 +356,46 @@ private void addExperimentNotesComponent() {
     content.add(experimentNotes);
   }
 
+  private void addSampleSourceInformationComponent() {
+    sampleSourceComponent.addClassName("sample-source-display");
+
+    content.add(sampleSourceComponent);
+  }
+
+  private Span createSampleSourceList(String title, Icon icon, List<String> tags) {
+    Span iconAndList = new Span();
+    iconAndList.addClassName("icon-with-list");
+    iconAndList.add(icon);
+    icon.addClassName("vaadin-icon");
```

Comment:
NIT: This is probably not needed. You can replace the CSS selector with the component id `vaadin-icon` instead of `.vaadin-icon` as vaadin introduces custom dom elements.